docker/candlepin-base-cs8/Dockerfile

```diff
@@ -24,5 +24,4 @@ RUN echo 'alias python="python3"' >> ~/.bashrc
 
 EXPOSE 8443 22
 
-#CMD ["/usr/bin/cp-test", "-t", "-u", "-r"]
-CMD ["true"]
\ No newline at end of file
+ENTRYPOINT ["/usr/sbin/init"]
```

Comment:
Is this change part of this task? if the cs8 container still uses supervisord then will this not break it?

Reply:
because how the containers are started i need to be still running after podma start command and this is doing that and now i would still add that because of the task migrate cs8 to systemd